app/helpers/application_helper.rb

```diff
@@ -194,7 +194,7 @@ def reaction_url(reaction)
     URL.reaction(reaction)
   end
 
-  def tag_url(tag, page)
+  def tag_url(tag, page = 1)
```

Comment:
tag_url is the method we call in the templates and so it makes sense to have the default parameter on this method instead of self.tag in the URL class

Reply:
I would keep the default in both places since the helper is only used in the views and the `URL` is only used in Ruby code